std_msgs/String str
---
nav_msgs/OccupancyGrid grid